x-pack/plugins/enterprise_search/public/applications/enterprise_search/cypress/integration/overview.spec.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../../../shared/cypress/commands';
+import { overviewPath } from '../../../shared/cypress/routes';
+
+context('Enterprise Search Overview', () => {
+  beforeEach(() => {
+    login();
+  });
+
+  it('should contain product cards', () => {
+    cy.visit(overviewPath);
+    cy.contains('Welcome to Elastic Enterprise Search');
+
+    cy.get('[data-test-subj="appSearchProductCard"]')
+      .contains('Launch App Search')
+      .should('have.attr', 'href')
+      .and('match', /app_search/);
+
+    cy.get('[data-test-subj="workplaceSearchProductCard"]')
+      .contains('Launch Workplace Search')
+      .should('have.attr', 'href')
+      .and('match', /workplace_search/);
+  });
+
+  it('should have a setup guide', () => {
+    // @see https://github.com/quasarframework/quasar/issues/2233#issuecomment-492975745
+    // This only appears to occur for setup guides - I haven't (yet?) run into it on other pages
+    cy.on('uncaught:exception', (err) => {
+      if (err.message.includes('> ResizeObserver loop limit exceeded')) return false;
+      return true; // re-throw other valid errors
+    });
```

Comment:
This was a fun (read: annoying) gotcha that it looks like the Security team also ran into it looks like: https://github.com/elastic/kibana/pull/58465

if we start running into this for more than just the setup guides, we should consider moving this to a shared `command.ts` file instead of just for a single test.

My best guess currently is that some component(s) trigger a ResizeObserver and some don't, but it's not clear to me which element(s) specifically.

Reply:
I think you're right! I _thought_ I saw it on a random Cypress example and copied it, but on second look/google I think I just made it up. I'll go ahead and lose the return true

```suggestion
    // @see https://github.com/quasarframework/quasar/issues/2233#issuecomment-492975745
    // This only appears to occur for setup guides - I haven't (yet?) run into it on other pages
    cy.on('uncaught:exception', (err) => {
      if (err.message.includes('> ResizeObserver loop limit exceeded')) return false;
    });
```